samples/cxx/gas_transport/gas_transport.cpp

```diff
@@ -13,11 +13,10 @@
 // This file is part of Cantera. See License.txt in the top-level directory or
 // at https://cantera.org/license.txt for license and copyright information.
 
-#include "cantera/thermo.h"
-#include "cantera/transport.h"
-#include "cantera/base/Solution.h"
+#include "cantera/core.h"
 #include "cantera/base/Array.h"
 
+#include <fstream>
```

Comment:
`fstream` is already included two lines below.

Reply:
whoops